lib/app.js

```diff
@@ -13,7 +13,7 @@
 
 
     function addCalculator() {
-      calculators.push(new Calculator());
+      calculators.push(" :) ");
```

Comment:
With this change, I can't remove calculators / todo lists any more. Why do you think that is?